web/template/layout/add-recipient-form.gohtml

```diff
@@ -0,0 +1,68 @@
+{{ define "add-recipient" }}
```

Comment:
since this will only ever be used by one template it would be better in the same file